script/test.py

```diff
@@ -1,42 +1,27 @@
-import argparse
 from selenium import webdriver
 from selenium.webdriver.common.keys import Keys
 from collections import namedtuple
-from selenium.webdriver.common.by import By
-from selenium.webdriver.support.ui import WebDriverWait
-from selenium.webdriver.support import expected_conditions as EC
-
+from collections import Counter
 
 
 LinkInfo  = namedtuple('LinkInfo', 'name href')
 
 
-def find_links(driver):  
-  links_elem = driver.find_elements_by_tag_name("a")    
-  return [LinkInfo(elem.text, elem.get_attribute('href')) for elem in links_elem]
-
-
-def print_link_info(links):
-  for value in links:    
-    print(value.name, value.href)
+def test_find_links(driver):
```

Comment:
If you want to count number of links then:
  * Use function from the script main.py
  * At the end of the function you should expression like: `assert found_number_of_links == expected_number_of_links`
  * Rename this function to `test_link_count`